pyproject.toml

```diff
@@ -23,9 +23,7 @@ select = ["ALL", "PT"]
 
 ignore = [
     # default
-    "ANN101", 
-    "ANN102", 
```

Comment:
`ruff` was updated to `v0.8`, and this warning came up:
```
warning: The following rules have been removed and ignoring them has no effect:
    - ANN101
    - ANN102
```

These were removed, and no issue.